### PR TITLE
82 update the channels page for the admin

### PR DESCRIPTION
**Pull request by EyosiSol**
- Created Date as A parameter was added 
- Searching and Filtering was added 
- Icon's automatically added based on the channel type 
- container hieght issues were fixed

### CODE REVIEW

**Comment by misudesu** [comment]
attach final image please!

File: components/shared/invite-admin-dialog.tsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    }
+  };
+
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const { name, value } = e.target;
+    setFormData((prev) => ({ ...prev, [name]: value }));
+  };
+
+  return (
+    <Dialog open={isOpen} onOpenChange={setIsOpen}>
```
Comment (misudesu) [inline components/shared/invite-admin-dialog.tsx:58]: this is what i recommend you , reuse it

File: components/shared/layout/sidebar.tsx [hunk 2]
```diff
@@ -13,11 +13,11 @@ import { cn } from "@/lib/utils";
 import { Button } from "@/components/ui/button";
 
 const menuItems = [
```
Comment (misudesu) [inline components/shared/layout/sidebar.tsx:15]: make it separate the menu and import it. we can manage this from one place. we have to do it, as easy to access

File: components/shared/mentors.tsx [hunk 3]
```diff
@@ -1,43 +1,22 @@
 "use client";
```
Comment (misudesu) [inline components/shared/mentors.tsx:1]: 👍awesome it is good to use as SSR

**Review by misudesu — CHANGES_REQUESTED** [review]

**Comment by EyosiSol** [comment]
image

**Comment by EyosiSol** [comment]
image

**Review by misudesu — APPROVED** [review]

**Pull request merged**